lambdas/url_access_patterns/lambda_for_generateURL/index.js

```diff
@@ -0,0 +1,48 @@
+const AWS = require("aws-sdk");
+const DT = require("date-and-time");
+const crypto = require("crypto");
+const dynamo = new AWS.DynamoDB.DocumentClient();
+const Hashids = require("hashids");
+
+function generateUniqueId() {
+  const salt = crypto.randomBytes(16).toString("base64");
+  const uniqueId = new Hashids(salt, 6);
+  const id = uniqueId.encode(Math.ceil(Math.random() * 1000));
+  return id;
+}
+function response(statusCode, message) {
+  return {
+    statusCode: statusCode,
+    body: JSON.stringify(message),
```

Comment:
No need to stringify in the case of `API gateway`

Reply:
forgot to change it. Anything else. @lakshya-20 ?